collector/fixtures/e2e-64k-page-output.txt

```diff
@@ -95,9 +95,6 @@ node_bcache_cache_misses_total{backing_device="bdev0",uuid="deaddd54-c735-46d5-8
 # HELP node_bcache_cache_read_races_total Counts instances where while data was being read from the cache, the bucket was reused and invalidated - i.e. where the pointer was stale after the read completed.
 # TYPE node_bcache_cache_read_races_total counter
 node_bcache_cache_read_races_total{uuid="deaddd54-c735-46d5-868e-f331c5fd7c74"} 0
-# HELP node_bcache_cache_readaheads_total Count of times readahead occurred.
```

Comment:
I think we should remove this from the tests and keep testing the case where `ps.CacheReadaheads != 0`

Reply:
If my understanding is correct, test cases has been updated to be used with `ps.CacheReadaheads != 0`. Also completed the DCO sign-off. Hope I didn't miss anything here.